extension/e2e-tests/test-fixtures-pt.ts

```diff
@@ -0,0 +1,84 @@
+import { test as base, chromium, BrowserContext, Page } from "@playwright/test";
+import path from "path";
+
+import { STELLAR_EXPERT_ASSET_LIST_JSON } from "./helpers/stubs";
+
+export const test = base.extend<{
+  context: BrowserContext;
+  extensionId: string;
+  page: Page;
+}>({
+  context: async ({}, use) => {
+    const pathToExtension = path.join(__dirname, "../build");
+    const context = await chromium.launchPersistentContext("", {
+      headless: false,
+      args: [
+        `--headless=new`,
+        `--disable-extensions-except=${pathToExtension}`,
+        `--load-extension=${pathToExtension}`,
+      ],
+    });
+
+    await use(context);
+    await context.close();
+  },
+  extensionId: async ({ context }, use) => {
+    // for manifest v2:
+    // let [background] = context.backgroundPages();
+    // if (!background) background = await context.waitForEvent("backgroundpage");
+
+    // for manifest v3:
+    let [background] = context.serviceWorkers();
+    if (!background) background = await context.waitForEvent("serviceworker");
+
+    const extensionId = background.url().split("/")[2];
+    await use(extensionId);
+  },
+  page: async ({ page }, use) => {
+    // Inject environment variable into browser context for memo validation bypass
+    await page.addInitScript(() => {
+      (window as any).IS_PLAYWRIGHT = "true";
+      try {
```

Comment:
this is pretty similar to the default `test-fixtures.ts`. Can we add the language setting as a config rather than creating  a separate fixture?